packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -6,7 +6,7 @@ import { getCFDAccountKey, isMobile, mobileOSDetect } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 import { getPlatformQRCode, PlatformsDesktopDownload, mobileDownloadLink } from '../Helpers/config';
 import { getTitle, platformsText, CTRADER_DESKTOP_DOWNLOAD } from '../Helpers/constants';
-import { CFD_PLATFORMS } from '../Helpers/cfd-config';
+import { CFD_PLATFORMS, MOBILE_PLATFORMS } from '../Helpers/cfd-config';
```

Comment:
@hamza-deriv please split this changes since it's outside of the src/features directory and need qa test

Reply:
Sure